setup.cfg

```diff
@@ -25,37 +25,38 @@ keywords = pytorch, deep learning, machine learning, remote sensing, satellite i
 
 [options]
 install_requires =
-    einops<0.5
-    # fiona 1.5+ required for fiona.transform module
-    fiona~=1.5
+    # einops 0.3+ required for einops.repeat
+    einops>=0.3,<0.5
+    # fiona 1.8+ required for reading empty files
+    fiona>=1.8,<2
     # kornia 0.6.4+ required for kornia.contrib.compute_padding
     kornia>=0.6.4,<0.7
-    matplotlib<4
-    numpy<2
+    # matplotlib 3.3+ required for (H, W, 1) image support in plt.imshow
+    matplotlib>=3.3,<4
+    # numpy 1.17.2+ required by pytorch-lightning
+    numpy>=1.17.2,<2
     # omegaconf 2.1+ required for to_object method
-    omegaconf~=2.1
-    # pillow 2.9+ required for height attribute
-    pillow>=2.9,<10
-    # Required to address protocolbuffers/protobuf#10051
-    protobuf<4
```

Comment:
No need to pin this here, it isn't used by CI so it won't cause us issues. If someone encounters an issue they can manually install an older protobuf.